src/common.h

```diff
@@ -13,8 +13,9 @@
 
 /* Constants */
 #define MVMU_DIM                        128
-#define N_CONSTANT_MVMUS_PER_CORE       6
-#define N_TRAINING_MVMUS_PER_CORE       2
+#define N_CONSTANT_MVMUS_PER_CORE       2
+#define N_TRAINING_MVMUS_PER_CORE       0
```

Comment:
```suggestion
#define N_CONSTANT_MVMUS_PER_CORE       6
#define N_TRAINING_MVMUS_PER_CORE       2
```
Just to reduce the number of changes.